.github/workflows/imagebuild.yml

```diff
@@ -0,0 +1,140 @@
+name: Build and Push Georust Docker Images
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+      - trying
+  pull_request:
+  merge_group:
+
+env:
+  LIBPROJ_VERSION: 9.4.0
+  BASE_IMAGE_NAME: libproj-builder
+
+jobs:
+  build_base_image:
+    name: Build the base image for geo and proj
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust_version: [1.77]
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
```

Comment:
Multi arch could be nice, but it's not something that we ever supported before AFAIK. I'd encourage you not to make multiarch a requirement for getting this merged.